### PR TITLE
Car table and counters

### PR DESCRIPTION
**Pull request by mariusz-kabala**

### CODE REVIEW

File: src/reducers/cars.js [hunk 1]
```diff
@@ -4,15 +4,15 @@ const initalStore = {
       name: 'Ford Ranger 3.2 TDCi Doppelkabine Limited',
       description: null,
       stockNumber: 'JP84315',
-      auctionEndTime: null,
+      auctionEndTime: new Date(Date.now() + 300000),
```
Comment (nainy-a1) [inline src/reducers/cars.js:7]: antd is using moment for dates, we might have some problems down the road

Comment (mariusz-kabala) [reply src/reducers/cars.js:7]: but I do not use this filed with antd

Comment (nainy-a1) [reply src/reducers/cars.js:7]: probably will be used if we save the car from the form to redux

Comment (mariusz-kabala) [reply src/reducers/cars.js:7]: aaa, oki, I will adjust it in the next PR

**Review by nainy-a1 — APPROVED** [review]

**Pull request merged**